src/shared/vscode/env.ts

```diff
@@ -58,6 +58,13 @@ export function isReleaseVersion(prereleaseOk: boolean = false): boolean {
     return (prereleaseOk || !semver.prerelease(extensionVersion)) && extensionVersion !== TEST_VERSION
 }
 
+/**
+ * Returns true when source mapping is available
+ */
+export function isSourceMappingAvailable(): boolean {
+    return extensionVersion === TEST_VERSION
+}
```

Comment:
For `npm run package -- --debug` builds we might want to scrape the stacktrace to see if it looks like it's non-minified. Eventually we should produce those debug builds as CI artifacts.

Reply:
Oh we don't even need to scrape anything. Webpack can inject a variable, I just didn't add it yet.